load_data_lambda/app.py

```diff
@@ -0,0 +1,8 @@
+from query.lambdas.load_data.load_data import LoadData
+from query.lib.common.logging import get_logger
+
+logger = get_logger(__name__)
+
+
+def load_data(event, context):
+    LoadData().query_service_data_load(event, context)
```

Comment:
I'd move the object created by `LoadData()` to the global scope to get the [global context performance optimizations](https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html).

Reply:
👍